migrate2rocky.sh

```diff
@@ -549,6 +568,28 @@ fix_efi () (
 	exit_message "Error updating uEFI firmware."
 )
 
+# Download and verify the Rocky Linux package signing key
+establish_gpg_trust () {
+    declare -g gpg_tmp_dir
+    gpg_tmp_dir=$(mktemp -d)
+    if [[ $? -ne 0 ]] || [[ -z "$gpg_tmp_dir" ]] || [[ ! -d "$gpg_tmp_dir" ]]; then
```

Comment:
There shouldn't be any case where mktemp exits with a 0 status but doesn't create the directory, so this should be fine for the above two lines:
`if ! gpg_tmp_dir=$(mktemp -d); then`